src/auth.nim

```diff
@@ -24,6 +25,11 @@ const
   }.toTable
 
 var
+  pool: HttpPool
```

Comment:
This can be removed, it's really only used to reduce latency for API requests

Reply:
i moved the http pool into a local variable